DataScience/FeatureImportance.py

```diff
@@ -114,7 +114,10 @@ def get_feature_importance(log_file, ml_args, warmstart_model=None, min_num_feat
 
     vw_inv_hash_cmd = vw_base + ' --invert_hash {0}'.format(invHash_fp)
     print('command to get invert hash file: ' + vw_inv_hash_cmd)
-    os.system(vw_inv_hash_cmd)
+    invhash_ret = os.system(vw_inv_hash_cmd)
+    if(invhash_ret != 0):
+        err_msg = "Error: VW failed to generate inverse hash."
+        sys.exit(err_msg)
```

Comment:
This gives a better error but doesn't address the underlying issue. Do we know why the model is invalid? Is there a way to prevent it?

Reply:
The model is passed as a parameter, so the call itself will determine its validity. I contemplated retrying without the model if the model cmd failed, but didn't want to introduce new functionality.